scipy/stats/tests/test_continuous_basic.py

```diff
@@ -654,3 +654,84 @@ def test_methods_with_lists(method, distname, args):
     npt.assert_allclose(result,
                         [f(*v) for v in zip(x, *shape2, loc, scale)],
                         rtol=1e-14, atol=5e-14)
+
+
+def test_moments_with_array_gh12192():
+    # array loc and scalar scale
+    vals0 = stats.norm.moment(n=1, loc=np.array([1, 2, 3]), scale=1)
+    expected0 = np.array([1., 2., 3.])
+    npt.assert_allclose(vals0, expected0, rtol=1e-20)
```

Comment:
Why rtol=1e-20? Anything less than np.finfo(float).eps ~1e-16 is requesting equality.

Reply:
Oh, I didn't know that. I will change it to equality then.